packages/core/src/components/forms/_input-group.scss

```diff
@@ -162,6 +165,30 @@ $input-button-height-large: $pt-button-height !default;
     }
   }
 
+  &.#{$ns}-small {
```

Comment:
please add support for this modifier in `_input.scss` as well.
this feature isn't complete till both input and InputGroup support it.

Reply:
I've added all the new css into _input.scss as well